### DISCUSSION TITLE
Mist页面模式左下角图标覆盖

### DISCUSSION THREAD

**Discussion by OnqiauJliu** [hugo-next/hugo-theme-next#107] [Q&A]
用Mist页面样式 左下角图标覆盖  这个地方在配置文件哪里调整呢？  我记得我没调整过这个地方
主题版本： v4.5.3
![image](https://github.com/hugo-next/hugo-theme-next/assets/60482496/6a06ecef-5de4-44e1-b5a0-a04af47801e3)

**Comment by elkan1788** [comment]
是的，这个是已知问题，只有自己一个在维护后面慢慢修复吧，现在主要在 `Gemini` 和 `Pisces` 两个模式上面，也期待后面有更多的开发者加入吧
